sycl/test-e2e/README.md

```diff
@@ -320,3 +320,23 @@ implementation header files is still in progress and the final set of these
 "fine-grained" includes that might be officially documented and suggested for
 customers to use isn't determined yet. **Until then, code outside of this project
 must keep using `<sycl/sycl.hpp>` provided by the SYCL2020 specification.**
+
+## Marking tests as expected to fail
```

Comment:
Should we do the same for the tests marked "UNSUPPORTED"?

Reply:
> Should we do the same for the tests marked "UNSUPPORTED"?

@uditagarwal97, the last sentence in the description answers this question. ;)

Reply:
Ahh.. I overlooked that. Time for me to get some coffee ☕